examples/vue/vue-storefront-2/pages/_path.vue

```diff
@@ -3,8 +3,8 @@
     <div>Hello world from your Vue project. Below is Builder Content:</div>
 
     <div v-if="canShowContent">
-      <div>page: {{ content.data.title }}</div>
-      <builder-render-content model="page" :content="content" />
+      <div>page: {{ (content && content.data && content.data.title) || 'Unpublished' }}</div>
+      <builder-render-content model="page" :content="content" :api-key="apiKey" />
```

Comment:
@steve8708 One breaking change to get symbols working with the new SDKs is that we now need the `apiKey` as an additional param. I need it so I can make the call to fetch the symbols in https://github.com/BuilderIO/builder/pull/951/files#diff-2c06c5c272e81428d9484d127d14481b913df0e6c8d9066671ee7bd8d38f0c05R19-R33